components/webac/src/main/java/org/trellisldp/webac/WebACService.java

```diff
@@ -133,7 +133,7 @@ public WebACService(final ResourceService resourceService,
      */
     public WebACService(final ResourceService resourceService,
             final CacheService<String, Set<IRI>> cache, final boolean checkMembershipResources) {
-        this.resourceService = requireNonNull(resourceService, "A non-null ResourceService must be provided!");
+        this.resourceService = resourceService;
         this.cache = requireNonNull(cache, "A non-null Cache must be provided!");
```

Comment:
I would take it out here too, or we'll just have to do it later.

Reply:
It's gone from there.

Reply:
Sorry, I meant `this.cache = requireNonNull(cache, "A non-null Cache must be provided!");`. Or is that not what you meant?

Reply:
Ah, okay. I'll take out that one too.